### PR TITLE
Expose a stopWatching method for createNodeishFsWithWatcher 

### PR DESCRIPTION
**Pull request by martin-lysk**
This PR exposes a "stopWatching" method for createNodeishFsWithWatcher to avoid passing in a AbortController. This way the function can keep track of its abortController and doesn't hit MaxListenersExceededWarning
`
(node:15668) MaxListenersExceededWarning: Possible EventTarget memory leak detected. 11 abort listeners added to [AbortSignal]. Use events.setMaxListeners() to increase limit`

### CODE REVIEW

File: pnpm-lock.yaml
Comment (jldec) [inline pnpm-lock.yaml:1]: @martin-lysk why include an updated lockfile in this PR?

**Review by jldec — CHANGES_REQUESTED** [review]

File: pnpm-lock.yaml
Comment (martin-lysk) [reply pnpm-lock.yaml:1]: thanks for the headsup - local pnpm version messed it up - fixed with last push

**Review by jldec — APPROVED** [review]
LGTM

**Comment by jldec** [comment]
Resolved https://github.com/opral/inlang-message-sdk/issues/57

**Pull request merged**